Standards/CCSMB-3.md

```diff
@@ -0,0 +1,22 @@
+# CCSMB-3: Directory
+
+*Author: Oliver Cooper (oeed)*
+*Last updated: 2017-01-27*
+*Version: v1.0.0*
```

Comment:
these should be spaced further apart - as-is they appear on the same line

Reply:
```suggestion
*Author: Oliver Cooper (oeed)*

*Last updated: 2017-01-27*

*Version: v1.0.0*
```

Fixed.